README.md

```diff
@@ -6,136 +6,145 @@
 
-A small sample is available as a playground.
+A small sample application is available as a playground.
 
 ### Instantiation
 
-#### iOS
+#### Android
 
-An example of instantiation as follow with the fallback to the pin code with some text descriptions to display for the user.
+Face authentication is only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricWeak)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service. Please note that if you are using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` Face authentication is only available on a Google Pixel 4 as of now.
 
-```
-var options = new BiometryOptions();
-options.LocalizedReasonBodyText = "REASON THAT APP WANTS TO USE BIOMETRY";
-options.LocalizedFallbackButtonText = "FALLBACK";
-options.LocalizedCancelButtonText = "CANCEL";
-
-// use LAPolicy.DeviceOwnerAuthenticationWithBiometrics for biometrics only with no fallback to passcode/password
-// use LAPolicy.DeviceOwnerAuthentication for biometrics+watch with fallback to passcode/password
-_biometryService = new BiometryService(options, async ct => "Biometrics_Confirm", LAPolicy.DeviceOwnerAuthentication);
+Please note that Encrypt/Decrypt methods are only available when using `.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)` in the `BiometricPrompt.PromptInfo.Builder` instantiation that is required for the service.
+
+Please also note that the title and subtitle are used for `Fingerprint` and `Face` biometry.
+
+Here is an example of instantiation of the service for Android.
+
+``` cs
+var promptBuilder = () => new BiometricPrompt.PromptInfo.Builder()
+	.SetTitle("Title")
+	.SetSubtitle("Subtitle")
+	.SetNegativeButtonText("Cancel")
+	.SetAllowedAuthenticators(AndroidX.Biometric.BiometricManager.Authenticators.BiometricStrong)
+	.Build();
+
+var biometryService = new BiometryService(
+	fragmentActivity: MainActivity.Instance,
+	promptInfoBuilder: promptBuilder,
+	loggerFactory: null
+);
 ```
 
-#### Android
+#### iOS
 
-Face ID is only available with the followwing configuration `.SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricWeak` during instantiation of the service.
+Please note that you must set `NSFaceIDUsageDescription` (key/value) in the `Info.plist` file otherwise the service may throw exceptions or fail.
 
-Encrypt/Decrypt method are only available with the following configuration `.SetAllowedAuthenticators(BiometricManager.Authenticators.BiometricStrong` during instantiation of the service.
+Please also note that the subtitle is used for `Fingerprint` biometry only.
```

Comment:
Do you meant to say that the `useOperationPrompt` property is only used for touchid?

Reply:
No I meant the subtitle that we pass to the builder, I will update the comment.